### PR TITLE
Fixes #427 :- Improves media wall cards design

### LINKED ISSUE #427: Improving media wall cards
Idea is to improve media wall cards with more focus on media and text. 

Mock Up 

screen shot 2017-06-26 at 4 00 18 am

@hemantjadon @mariobehling Please suggest.

### PR DESCRIPTION
**Pull request by Achint08**
**Changes proposed in this pull request**
- Implemented a new design for media wall cards.
- Defined columns for all type of screen-sizes
- Changed background color

**Screenshots** 

screen shot 2017-06-27 at 3 54 40 am

**Link to live demo:** 
https://achint08.github.io/loklak_search/

**Closes #427 **

Please review @mariobehling @hemantjadon @vibhcool @singhpratyush @djmgit @kavithaenair

### CODE REVIEW

**Comment by Achint08** [comment]
@hemantjadon Checkout now. :)

**Comment by Achint08** [comment]
@singhpratyush I am looking into cards flickering issue now. Just wait. 

@hemantjadon I think some problem is occurring due to font units on larger screen. Just wait for the next commit.

**Comment by Achint08** [comment]
@hemantjadon @singhpratyush Please review now.
Actually, I have added `scale` on hovering. However, it is causing the flickering issue. Will look out for some solution and make a later PR. 
Otherwise, all issues are solved now.
> The cards flicker while moving from other sections (like the header) to card section.
This problem was occurring because of the `last-child` element I used somewhere. Check out now. https://achint08.github.io/loklak_search/

**Comment by singhpratyush** [comment]
@Achint08: The flickering looks like too much. I think it would be better to fix it here before deploying this.

**Comment by Achint08** [comment]
@singhpratyush Can you specify the browser?

**Comment by singhpratyush** [comment]
@Achint08: It is Google Chrome 59.0.3071.86 (Official Build) (64-bit).

**Comment by Achint08** [comment]
@singhpratyush Removed `scale`Will make a PR later for transition

**Comment by Achint08** [comment]
@singhpratyush Solved. It was because I forgot to specify `z-index` .

File: src/app/media-wall/media-wall.component.scss [hunk 1]
```diff
@@ -15,53 +20,78 @@ body {
...
 	}
 }
 
-@media only screen and (max-width: 600px) {
+@media only screen and (min-width: 800px) and (max-width: 1200px) {
+	:root {
+		font-size: 16px;
+	}
   .masonry {
-  	column-count: 1;
```
Comment (hemantjadon) [inline src/app/media-wall/media-wall.component.scss]: Indentation in this block

File: src/app/media-wall/media-wall.component.scss [hunk 2]
```diff
@@ -15,53 +20,78 @@ body {
...
+  	column-count: 3;
+  	column-gap: 0;
+	}
+}
+
+@media only screen and (min-width: 1200px) and (max-width: 1500px) {
+	:root {
```
Comment (hemantjadon) [inline src/app/media-wall/media-wall.component.scss]: here also

**Comment by Achint08** [comment]
@hemantjadon Done!

**Pull request merged**